app/src/main/res/values/strings.xml

```diff
@@ -255,65 +254,65 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="quest_wheelchairAccess_location_name_title">Is the location %s wheelchair accessible?</string>
     <string name="quest_wheelchairAccess_limited">Partially</string>
     <string name="quest_wheelchairAccess_limited_description_business">Partially means that there is no more than a single step to access the business and that most rooms inside are wheelchair accessible.</string>
     <string name="quest_wheelchairAccess_limited_description_public_transport">Partially means that there is no more than a single step to access it.</string>
-    <string name="quest_wheelchairAccess_description_toilets">A wheelchair sign like this might be visible if the toilets are accessible.
-    Partially means that a wheelchair can enter and use the toilets, but no handrail or adapted hand basin is present.</string>
+    <string name="quest_wheelchairAccess_description_toilets">A wheelchair sign like this might be visible if the restroom is accessible.
+    Partially means that a wheelchair can enter and use the restroom, but no handrail or adapted hand basin is present.</string>
     <string name="quest_recycling_type_title">What type of recycling facility is this?</string>
-    <string name="recycling_centre">Centre</string>
+    <string name="recycling_centre">Center</string>
     <string name="overground_recycling_container">Container</string>
     <string name="underground_recycling_container">Underground container</string>
     <string name="quest_generic_error_a_field_empty">You left a field empty</string>
-    <string name="quest_streetName_menuItem_language_simple">%1$s - %2$s</string>
-    <string name="quest_streetName_menuItem_language_native">%1$s - %2$s (%3$s)</string>
-    <string name="quest_streetName_answer_cantType">"Can't type the characters on the sign"</string>
+    <string name="quest_streetName_menuItem_language_simple">%1$s – %2$s</string>
+    <string name="quest_streetName_menuItem_language_native">%1$s – %2$s (%3$s)</string>
+    <string name="quest_streetName_answer_cantType">"Can’t type the characters on the sign"</string>
     <string name="quest_streetName_cantType_title">Missing keyboard layout?</string>
     <string name="quest_streetName_cantType_open_settings">Open settings</string>
     <string name="quest_streetName_cantType_open_store">Visit app store</string>
     <string name="quest_streetName_cantType_description">"If you use a default keyboard, chances are that you can simply enable the keyboard layout for the language you need in the input method settings.
 
-Otherwise, you can download another keyboard in the app store. Popular keyboards that support many languages are Google Gboard, SwiftKey Keyboard and Multiling O Keyboard."</string>
+Otherwise, you can download another keyboard in the app store. Popular keyboards that support many languages are Google Gboard, SwiftKey Keyboard, and Multiling O Keyboard."</string>
```

Comment:
Sure about this?

Reply:
Yes, US English use the listing comma (I’ve removed it in the en-GB translation).

Reply:
OK, merging!